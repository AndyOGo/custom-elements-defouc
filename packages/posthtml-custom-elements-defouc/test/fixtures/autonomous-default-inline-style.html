<custom-element>content</custom-element>
