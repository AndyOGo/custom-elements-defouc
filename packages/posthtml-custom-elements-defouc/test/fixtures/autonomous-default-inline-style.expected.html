<custom-element style="visibility: hidden">content</custom-element>
